tests/snapshots.rs

```diff
@@ -580,7 +580,7 @@ fn convert_spv(name: &str, adjust_coordinate_space: bool, targets: Targets) {
     check_targets(&module, name, targets);
     naga::valid::Validator::new(
         naga::valid::ValidationFlags::all(),
-        naga::valid::Capabilities::empty(),
+        naga::valid::Capabilities::all(),
```

Comment:
I'm a little uncomfortable changing the conditions under which we're validating the other tests. Let's have another `bool` argument to `convert_spv` instead. Or, ideally, combine `adjust_coordinate_space` and the new bool into a `bitflags!` type, so that we use meaningful names in the call sites instead of `false, true`, etc.

Reply:
Actually, as it turns out, those new tests don't need any particular capabilities - I think they needed them for a while during development, when I accidentally broke uniformity analysis for some opcodes 🙃 

Anyway, I've rolled back to `Capabilities::empty()` and everything seems to be passing.